src/utils/api.js

```diff
@@ -5,16 +5,30 @@ const inquirer = require('inquirer')
 
 const creds = require('./creds')
 const getQuestions = require('./get-questions')
-const { LOGIN_URL, SIGNUP_URL, API_URL } = require('../constants')
+const {
+  EU_API_URL, EU_LOGIN_URL, EU_SIGNUP_URL, US_API_URL, US_LOGIN_URL, US_SIGNUP_URL
+} = require('../constants')
 
 module.exports = {
   accessToken: '',
   spaceId: null,
+  region: 'EU',
+
+  setRegion (region) {
+    this.region = region && region.toLowerCase()
+    if (this.region) {
+      console.log(chalk.green('✓') + ' Setting region to: ', this.region.toUpperCase() || 'EU')
+    }
+
+    this.API_URL = this.region === 'us' ? US_API_URL : EU_API_URL
+    this.LOGIN_URL = this.region === 'us' ? US_LOGIN_URL : EU_LOGIN_URL
+    this.SIGNUP_URL = this.region === 'us' ? US_SIGNUP_URL : EU_SIGNUP_URL
```

Comment:
Like i mentioned above, you don't need to change the `login/signup/logout` region